entries/d/deltadental.com.json

```diff
@@ -0,0 +1,51 @@
+{
+  "Delta Dental": {
+    "domain": "deltadental.com",
+    "img": "deltadental.com.png",
+    "tfa": [
+      "sms"
+    ],
+    "additional-domains": [
+      "deltadentalcoversme.com",
+      "deltadentalar.com",
+      "deltadentalks.com",
+      "deltadentalok.org",
+      "deltadentalpr.com",
+      "kentucky.deltadental.com",
+      "northcarolina.deltadental.com",
+      "tn.deltadental.com",
+      "www.deltadentalaz.com",
+      "www.deltadentalco.com",
+      "www.deltadentalct.com",
+      "www.deltadentalia.com",
```

Comment:
Can you confirm that these work?
I'm apparently blocked from accessing them.

Reply:
Yeah those two resolve for me. Maybe you can use a US-based VPN or something to see them?

Reply:
I tried but they seem to block VPNs. If they work for you then it's fine.